src/teleop/jetson/jetson_teleop.py

```diff
@@ -97,37 +100,108 @@ def __init__(self):
             "finger",
             "gripper",
         ]
+        SA_NAMES = ["joint_a", "joint_b", "joint_c", "joint_d", "scoop", "microscope"]
+
         self.ra_cmd = JointState(
-            name=[name for name in self.ra_names],
-            position=[math.nan for i in range(len(self.ra_names))],
-            velocity=[0.0 for i in range(len(self.ra_names))],
-            effort=[math.nan for i in range(len(self.ra_names))],
+            name=[name for name in RA_NAMES],
+            position=[math.nan for i in range(len(RA_NAMES))],
+            velocity=[0.0 for i in range(len(RA_NAMES))],
+            effort=[math.nan for i in range(len(RA_NAMES))],
+        )
+
+        self.sa_cmd = JointState(
+            name=[name for name in SA_NAMES],
+            position=[math.nan for i in range(len(SA_NAMES))],
+            velocity=[0.0 for i in range(len(SA_NAMES))],
+            effort=[math.nan for i in range(len(SA_NAMES))],
         )
 
-    def ra_control_callback(self, msg):
+    def filter_xbox_axis(
+        self,
+        axes_array: "list[float]",
+        axis_name: str,
+        deadzone_threshold: float,
```

Comment:
Seems like it would be worth having a default for this. Could be a constant defined towards the top of the file or yaml